### PR TITLE
Added french translations

### PR DESCRIPTION
**Pull request by gbastien**
Hi,

could you please review and merge if everything is correct?

We are integrating this very adaptable package in our projects, we will come with other PRs soon and see if it fits your needs or if we override it locally.

Thank you,

Gauthier

### CODE REVIEW

File: ftw/labels/locales/fr/LC_MESSAGES/ftw.labels.po [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+#. Default: "Labels"
+#: ./ftw/labels/portlets/labeljar.pt:9
+msgid "labeljar_portlet_heading"
+msgstr "Étiquettes"
+
+#. Default: "Please choose a title."
+#: ./ftw/labels/browser/labelsjar.py:22
+msgid "lable_title_is_missing"
+msgstr "Encodez un titre."
+
```
Comment (njohner) [inline ftw/labels/locales/fr/LC_MESSAGES/ftw.labels.po]: "Veuillez choisir un titre pour l'étiquette"

File: ftw/labels/locales/fr/LC_MESSAGES/ftw.labels.po [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+#. Default: "Create label"
+#: ./ftw/labels/portlets/labeljar.pt:53
+msgid "button_create_label"
+msgstr "Créer une étiquette"
+
+#. Default: "Delete label"
+#: ./ftw/labels/browser/templates/edit_label.pt:55
+msgid "button_delete_label"
+msgstr "Supprimer"
```
Comment (njohner) [inline ftw/labels/locales/fr/LC_MESSAGES/ftw.labels.po]: "Supprimer l'étiquette"

File: ftw/labels/locales/sync_pos.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+files="ftw.labels"
+languages="de fr"
+
+for file in $files; do
+    for language in $languages; do
+        i18ndude sync --pot $file.pot $language/LC_MESSAGES/$file.po
+        msgfmt -o $language/LC_MESSAGES/$file.mo $language/LC_MESSAGES/$file.po
+    done
+done
```
Comment (njohner) [inline ftw/labels/locales/sync_pos.sh]: We use `bin/i18n-build` to manage the translations.

**Comment by njohner** [comment]
I requested changes just to be consistent with the German text.

**Comment by gbastien** [comment]
Hi @njohner 

thank you for review, I updated the translations and removed the sync_pos.sh script.

Gauthier

**Comment by njohner** [comment]
@gbastien Can you capitalize "french" in the changelog?
Then I'll merge and make a release.

**Comment by gbastien** [comment]
Hi @njohner 

done!  Could you please wait for a new release, I would like to propose a second little PR, it is about redirecting to HTTP_REFERER instead absolute_url in the @@labels-jar view, _redirect method...

I do the second PR when this one is merged.

Thank you,
Gauthier

**Comment by gbastien** [comment]
Hi @njohner 
I come back to you regarding this PR, is it ready to be merged?  So I can propose 2 others small PRs ;-)
Thank you,
Gauthier

**Comment by njohner** [comment]
Hi @gbastien,
Sorry, here you go.

N.

**Pull request merged**